multiqc/modules/seqyclean/seqyclean.py

```diff
@@ -0,0 +1,123 @@
+			PE2ReadswContam_found = RowTwo.split("\t")[45]
+			PE2DiscByContam = RowTwo.split("\t")[55]
+			PE2DiscByLength = RowTwo.split("\t")[56]
+
+			#add the variables above into a dictionary for the total values table
+			self.data_total.update({
+				f['s_name']: {
+					'PairsKept': PairsKept,
+					'PairsDiscarded': PairsDiscarded
+				}
+			})
+			#add the above variables into a dictionary for a breakdown table
+			self.data_breakdown.update({
+				f['s_name']: {
+					'PE1ReadsAn':PE1ReadsAn,
+					'PE1TruSeqAdap_found':PE1TruSeqAdap_found,
+					'PE1ReadswVector_found':PE1ReadswVector_found,
+					'PE1ReadswContam_found':PE1ReadswContam_found,
+					'PE1DiscByContam':PE1DiscByContam,
+					'PE1DiscByLength':PE1DiscByLength,
+					'PE2ReadsAn':PE2ReadsAn,
+					'PE2TruSeqAdap_found':PE2TruSeqAdap_found,
+					'PE2ReadswVector_found':PE2ReadswVector_found,
+					'PE2ReadswContam_found':PE2ReadswContam_found,
+					'PE2DiscByContam':PE2DiscByContam,
+					'PE2DiscByLength':PE2DiscByLength
+				}
+			})
+		
+		# Write the results to a file
+		self.write_data_file(self.data_total, 'seqyclean_results')
+		self.write_data_file(self.data_breakdown, 'seqyclean_results')
+		#headers for the "general information" at the top of the Multiqc Analysis
+		self.seqyclean_general_stats_table()
+		#These functions create the sections in the report summary
+		firstkeys = ['PairsKept', 'PairsDiscarded']
+		secondkeys =['PE1ReadsAn','PE1TruSeqAdap_found','PE1ReadswVector_found','PE1ReadswContam_found','PE1DiscByContam','PE1DiscByLength','PE2ReadsAn','PE2TruSeqAdap_found','PE2ReadswVector_found','PE2ReadswContam_found','PE2DiscByContam','PE2DiscByLength']
+		pconfig = {
+            'id': 'seqyclean_assignment_plot',
+            'title': 'Seqyclean: Reads',
+            'ylab': 'Num of Reads',
+        }
+		config = {
+            'id': 'seqyclean_assignment_plot_2',
+            'title': 'Seqyclean: Breakdown Reads',
+            'ylab': 'Num of Reads',
+        }
+		self.add_section (
+			name = 'Total Seqyclean Results',
+			anchor = 'seqyclean-first',
+			description = 'This shows the results of the seqyclean process',
+			helptext = "If you're not sure _how_ to interpret the data, we can help!",
+			plot = bargraph.plot(self.data_total, firstkeys, pconfig)
+		)
+		self.add_section (
+			name = 'Seqyclean Results Breakdown',
+			anchor = 'seqyclean-second',
+			description = 'This shows the breakdown results of the seqyclean process',
+			plot = bargraph.plot(self.data_breakdown, secondkeys, config)
+		)
+				
+	def seqyclean_general_stats_table(self):
+		""" Take the parsed stats from the sargasso report and add them to the
+		basic stats table at the top of the report """
+		headers = OrderedDict()
+		headers['PairsKept'] = {
+			'title': 'PairsKept: Seqyclean pairs that were preserved',
+			'description': 'Num of pairs kept from seqyclean analysis',
+			'scale': 'YlGn',
+			'format': '{} bp',
+			'ylab': '# Reads',
```

Comment:
I don't think that `ylab` does anything in a table...

Also, please use `config.read_count_multiplier` and associated variables for read count columns - see the docs: https://multiqc.info/docs/#step-3-adding-to-the-general-statistics-table

```python
'title': '{} Reads'.format(config.read_count_prefix),
'description': 'Number of reads ({})'.format(config.read_count_desc),
'modify': lambda x: x * config.read_count_multiplier,
```

Reply:
I think I have made the appropriate edits.